### PR TITLE
[STACK 1] Fix `Delivery.from_mosaiq`, remove db repeat

### PR DESCRIPTION
**Pull request by SimonBiggs**
@dg1an3 here I removed that db-repeat that I was doing. I highly suspect it was because I hadn't undergone the ordering appropriately as you highlighted. It does align with the bug I was seeing. And, now we have a test to verify that it is appropriately and consistently pulling from MSSQL. I'd rather that test throw an error if it doesn't match baseline on first db query. So, I removed the retry loop. Thanks for helping me go through the process of pulling out that nasty thorn.

### CODE REVIEW

File: lib/pymedphys/_mosaiq/delivery.py [hunk 1]
```diff
@@ -273,128 +264,77 @@ def delivery_data_sql(connection, field_id):
...
-
-    @classmethod
-    def _from_mosaiq_base(cls, connection, field_id):
-        txfield_results, txfieldpoint_results = fetch_and_verify_mosaiq_sql(
-            connection, field_id
-        )
-
-        total_mu = np.array(txfield_results[0]).astype(float)
-        cumulative_percentage_mu = txfieldpoint_results[:, 0].astype(float)
+        cumulative_percentage_mu = tx_field_points["Index"].to_numpy(dtype=float)
```
Comment (sjswerdloff) [inline lib/pymedphys/_mosaiq/delivery.py:337]: If this is from TxFieldPoints You need to be careful to renormalize against the last  value.  Some conventional plans can use 1.0 as the final cumulative meterset weight.  While you aren't trying to deal with Particle Therapy those can be absolute meterset or something rather arbitrary. This is probably OK for ELEKTA linacs and Monaco as the TPS, but formally, it's not guaranteed to be %

**Review by sjswerdloff — APPROVED** [review]
Consider renormalizing TxFieldPoints [index] column contents

**Review by dg1an3 — APPROVED** [review]

File: lib/pymedphys/_mosaiq/delivery.py (hunk 1 above)
Comment (SimonBiggs) [reply lib/pymedphys/_mosaiq/delivery.py:337]: Heya @msobolewski,

Can I give a shout-out to the value of peer review (and the value of having @sjswerdloff on board) for this one. Every time I've seen this that last point has always been 1.0, so I assumed that is how it was. However, potentially a Monaco upgrade (maybe) or a different TPS, would have completely thrown that assumption asunder and we would have been reporting total MU/cumulative MU as way off.

---

Thanks @sjswerdloff for this 🙂, massively appreciated.

Comment (SimonBiggs) [reply lib/pymedphys/_mosaiq/delivery.py]: Fixed in 4da5a90a4dbd3e6d58e2660dfb7b2beeb0fa76a2

**Pull request merged**